docs/getting_started.md

```diff
@@ -194,20 +194,19 @@ kind: Cluster
 metadata:
   name: prod-workload
 spec:
-    clusterNetwork:
-        services:
-            cidrBlocks: ["100.64.0.0/13"]
-        pods:
-            cidrBlocks: ["100.96.0.0/11"]
-        serviceDomain: "cluster.local"
-    providerSpec:
-      value:
-        apiVersion: "vsphereproviderconfig/v1alpha1"
-        kind: "VsphereClusterProviderConfig"
-        vsphereUser: "<REDACTED>"
-        vspherePassword: "<REDACTED>"
-        vsphereServer: "<REDACTED>"
-        vsphereCredentialSecret: ""
```

Comment:
Can we keep the  `vsphereCredentialSecret` field in the doc?

Reply:
Hi @sidharthsurana,

I removed it because we temporarily removed the field from the cluster provider spec. I have another branch I'm preparing that augments credentials by allowing you to initialize one or more credentials structs that then get turned into secrets on the management cluster (and eventually the workload clusters).

Reply:
But I can put it back here so that it's only removed once the other stuff is ready.

Reply:
Actually, I'm going to leave this out. The field would be omitted when marshaled anyway as it is empty. This is not an example of all possible fields, but the example of what happens when the manifests are created.

We cannot expect every manifest to include every possible configuration. The k/k community has routinely decided that GoDocs are the means by which people can discover additional field information.

Eventually there will be better documentation for how to use credential secrets.

Reply:
Hi @frapposelli,

Just getting your opinion here. There's just zero reason for the secret before the management cluster exists anyway. So I don't really think it belongs in any of our generated data since that's all generally for the pivot, and if people know how to create secrets, then they know how to find the docs on how to use them IMO.

And as I said, I'm working on a refactor to allow for a single credentials manager defined as part of the cluster spec to be used by both the cloud providers as well as the machine specs (they can select which secret to use in order to partition creation of machines in different locations)